### PR TITLE
Increase code coverage: post-911-gib-status

### PR DESCRIPTION
**Pull request by nicksayre**
## Summary

- Adds 3 tests to bring coverage from 88% to 100% for this file: `vets-website/src/applications/post-911-gib-status/actions/post-911-gib-status.js`.
- I work for VA IIR.

## Related issue(s)

Team IIR's ticket: https://app.zenhub.com/workspaces/va-iir-6508c0bd79e64e0fb5855caf/issues/gh/department-of-veterans-affairs/va-iir/487

## Testing done

From `vets-website` do `yarn test:coverage-app post-911-gib-status`. The console indicate passing tests, and none failing. A browser window is launched with a coverage report. Click on the "actions" link and you will see a readout of the file in question.

- Added a test to cover 403 response.
- Added a test to cover 404 response.
- Added a test to cover situation where an error is returned with no error status codes.

## Screenshots

Screenshot 2024-04-03 at 9 54 49 AM

## What areas of the site does it impact?

No other areas are affected.

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### CODE REVIEW

File: src/applications/post-911-gib-status/tests/actions/post-911-gib-status.unit.spec.js [hunk 1]
```diff
@@ -4,15 +4,17 @@ import {
   mockFetch,
   setFetchJSONFailure,
   setFetchJSONResponse,
-} from 'platform/testing/unit/helpers';
+} from '@department-of-veterans-affairs/platform-testing/helpers';
 import {
```
Comment (nicksayre) [inline src/applications/post-911-gib-status/tests/actions/post-911-gib-status.unit.spec.js:8]: Linting issue fixed.

File: src/applications/post-911-gib-status/tests/actions/post-911-gib-status.unit.spec.js [hunk 2]
```diff
@@ -132,7 +134,7 @@ describe('getEnrollmentData', () => {
   });
 
   it('dispatches GET_ENROLLMENT_DATA_FAILURE on unexpected error without code', done => {
-    setFetchJSONFailure(global.fetch.onCall(0), Promise.reject({}));
+    setFetchJSONFailure(global.fetch.onCall(0), Promise.reject(new Error()));
     const thunk = getEnrollmentData();
```
Comment (nicksayre) [inline src/applications/post-911-gib-status/tests/actions/post-911-gib-status.unit.spec.js:138]: Linting issue fixed.

**Review by rmessina1010 — APPROVED** [review]
LGTM

**Review by john-rodriguez-adhocteam — APPROVED** [review]
Code LGTM and I got the tests passing locally and see the report of 100%. Awesome work!

**Pull request merged**